3scale/porta
# THREESCALE-7955: Fix updating user custom fields in the admin portal UI

**What this PR does / why we need it**:

Fix updating custom user fields from the admin portal.

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-7955

**Verification steps** 

- Create a custom field for the User model via Field Definitions
- Try to update the value of this field in the admin portal User edit form
- Notice that the field's value is updated properly

**Special notes for your reviewer**:
